src/Tizen.Applications.Common/Tizen.Applications/AppControl.cs

```diff
@@ -640,6 +638,99 @@ public static void SendTerminateRequest(AppControl terminateRequest)
+        /// The operation is mandatory information for the launch request.
+        /// If the operation is not specified, AppControlOperations.Default is used by default.
+        /// If the operation is AppControlOperations.Default, the application ID is mandatory to explicitly launch the application.<br/>
+        /// Since Tizen 2.4, the launch request of the service application over out of packages is restricted by the platform.
+        /// Also, implicit launch requests are NOT delivered to service applications since 2.4.
+        /// To launch a service application, an explicit launch request with the application ID given by property ApplicationId MUST be sent.
+        /// </remarks>
+        /// <param name="launchRequest">The AppControl.</param>
+        /// <param name="replyAfterLaunching">The callback function to be called when the reply is delivered.</param>
+        /// <exception cref="ArgumentException">Thrown when failed because of the argument is invalid.</exception>
+        /// <exception cref="Exceptions.AppNotFoundException">Thrown when the application to run is not found.</exception>
+        /// <exception cref="Exceptions.LaunchFailedException">Thrown when the request failed to launch the application.</exception>
+        /// <exception cref="Exceptions.LaunchRejectedException">Thrown when the launch request is rejected.</exception>
+        /// <exception cref="Exceptions.OutOfMemoryException">Thrown when the memory is insufficient.</exception>
+        /// <exception cref="Exceptions.PermissionDeniedException">Thrown when the permission is denied.</exception>
+        /// <exception cref="TimeoutException">Thrown when failed because of timeout.</exception>
+        /// <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+        /// <since_tizen> 6 </since_tizen>
+        public static async Task<AppControlResult> SendLaunchRequestAsync(AppControl launchRequest, AppControlReplyCallback replyAfterLaunching)
+        {
+            if (launchRequest == null)
+            {
+                throw new ArgumentNullException(nameof(launchRequest));
+            }
+
+            return await Task.Run(() =>
+            {
+                AppControlResult appControlResult = AppControlResult.None;
+                Interop.AppControl.ErrorCode err;
+
+                Interop.AppControl.ResultCallback cb = (IntPtr handle, int result, IntPtr userData) =>
+                {
+                    appControlResult = (AppControlResult)result;
+                };
+
+                if (replyAfterLaunching != null)
+                {
+                    int requestId = 0;
+                    lock (s_replyNativeCallbackMaps)
+                    {
+                        requestId = s_replyNativeCallbackId++;
+                        s_replyNativeCallbackMaps[requestId] = replyAfterLaunching;
+
+                        s_replyNativeCallback = (launchRequestHandle, replyRequestHandle, result, userData) =>
```

Comment:
Why do you re-create a callback for each request?
I recommand to create a s_replayNativeCallback in static constructor.